doc/man3/EVP_PKEY_fromdata.pod

```diff
@@ -50,9 +61,93 @@ EVP_PKEY_fromdata() return 1 for success and 0 or a negative value for
 failure.  In particular a return value of -2 indicates the operation is
 not supported by the public key algorithm.
 
+=head1 EXAMPLES
+
+=head2 Creating an RSA keypair
+
+ #include <openssl/evp.h>
+
+ /*
+  * These are extremely small to make this example simple.  A real
+  * and secure application will not use such small numbers.  A real
+  * and secure application is expected to use BIGNUMs, and to build
+  * this array dynamically.
+  */
+ const unsigned long rsa_n = 0xbc747fc5;
```

Comment:
If we are going to do this it should probably be a real case using proper BN instead of this invalid key data..

I think we also need to make the param_builder public..

Reply:
I don't think it has to be valid data for this example. What will the input be? Will that add a function that builds BNs from raw data? That's extra stuff that's going to be distracting more than anything else.

We could consider a more complete cookbook, but that should be elsewhere.

Reply:
Ok -  keep a mental note maybe that we should create a link here to the more complete one..

Reply:
I could add a comment saying so

Reply:
I also added a bit of introdductionary text at the beginning of the EXAMPLES section:

> These examples are very terse for the sake of staying on topic, which
are the EVP_PKEY_fromdata() set of functions.  In real applications,
BIGNUMs would be handled, and converted to byte arrays with
BN_bn2nativepad(), but that's off topic here.